interfaces/builtin/steam_support.go

```diff
@@ -138,8 +138,25 @@ mount options=(rw, rbind) /oldroot/home/{,**} -> /newroot/home/{,**},
 mount options=(rw, rbind) /oldroot/snap/** -> /newroot/snap/**,
 mount options=(rw, rbind) /oldroot/home/**/usr/ -> /newroot/usr/,
 mount options=(rw, rbind) /oldroot/home/**/usr/etc/** -> /newroot/etc/**,
+
+mount options=(rw, rbind) /oldroot/media/**/steamapps/common/**/usr/ -> /newroot/usr/,
+mount options=(rw, rbind) /oldroot/mnt/**/steamapps/common/**/usr/ -> /newroot/usr/,
+mount options=(rw, rbind) /oldroot/run/media/**/steamapps/common/**/usr/ -> /newroot/usr/,
+mount options=(rw, rbind) /oldroot/opt/**/steamapps/common/**/opt/ -> /newroot/opt/,
+mount options=(rw, rbind) /oldroot/srv/**/steamapps/common/**/srv/ -> /newroot/srv/,
```

Comment:
If your Steam library is in `/opt` or `/srv` then you'd need this to be more like

```suggestion
mount options=(rw, rbind) /oldroot/opt/**/steamapps/common/**/usr/ -> /newroot/usr/,
mount options=(rw, rbind) /oldroot/srv/**/steamapps/common/**/usr/ -> /newroot/usr/,
```

and you'd also need equivalent rules for `.../usr/etc/ -> /newroot/etc/`.

It might make more sense to widen these to be more like:

```
mount options=(rw, rbind) /oldroot/{home,media,mnt,run/media,opt,srv}/**/steamapps/common/** -> /newroot/**,
```

to make this profile less dependent on implementation details of how pressure-vessel works, which can change at any time. That would make the separate rules for `usr/etc`, `ld.so.cache`, etc. unnecessary.

Reply:
https://github.com/snapcore/snapd/pull/13053/commits/b15d3b131b57eb42f42aed8608b8f90991d40e4c I opted for the more broad route for now